panel/wrappers/ipywidget.py

```diff
@@ -0,0 +1,303 @@
+"""Functionality to enable easy interaction with Traitlets models and ipywidgets via familiar APIs from Param like watch, bind, depends, and rx.
```

Comment:
I would not put the ipywidget wrapping functionality in an `ipywidget` file in the root folder because panels root `__init__` already imports `ipywidget` from `panel.io`.